core/controllers/admin.py

```diff
@@ -1008,3 +1009,16 @@ def put(self):
         user_services.log_username_change(
             self.user_id, old_username, new_username)
         self.render_json({})
+
+
+class NumberOfDeletionRequestsHandler(base.BaseHandler):
+    """Handler for deleting account via admin page."""
```

Comment:
I don't think this handler does what you say it does...

Reply:
Done.